### ISSUE TITLE
`SelectedListValue` is not referenced

### ISSUE THREAD

**Issue by glen-84** [graphql/composite-schemas-spec#76]
There's a `SelectedListValue` node, but it doesn't show where it's used. Is it part of `SelectedValue`, like `SelectedObjectValue` is?

```
SelectedValue ::
    Path
    SelectedObjectValue
    Path . SelectedObjectValue
    Path SelectedListValue            # Add this? (but not SelectedListValue on its own)
    SelectedValue | SelectedValue
```

The description is also a bit confusing:

> A SelectedListValue is an ordered list of SelectedValue wrapped in square brackets []. It is used to express semantic equivalence between an argument expecting a list of values and the values of a list field within the output object.
>
> The SelectedListValue differs from the ListValue defined in the GraphQL specification by only allowing one SelectedValue as an element.

It's an "ordered list" ... but only allows one value. Is it really a list then?

**Comment by PascalSenn** [comment]
@glen-84  I agree, the text is confusing. I think it's historical that we first wanted to allow for more than one item.
